src/state/airdrop/selectors.ts

```diff
@@ -1,10 +1,29 @@
+import { differenceBy } from 'lodash';
 import { createSelector } from 'reselect';
 
+import { Wallet } from 'app/consts';
+import { HDSegwitP2SHAirWallet } from 'app/legacy';
 import { ApplicationState } from 'app/state';
 
+import { WalletsState } from '../wallets/reducer';
 import { AirdropState } from './reducer';
 
 const local = (state: ApplicationState): AirdropState => state.airdrop;
+const walletsState = (state: ApplicationState): WalletsState => state.wallets;
 
 export const thankYouSeen = createSelector(local, state => state.thankYouSeen);
 export const thankYouFlowCompleted = createSelector(local, state => state.thankYouFlowCompleted);
+export const airdropReadyWallets = createSelector(walletsState, state =>
```

Comment:
to delete, will be unused

Reply:
used in line 48 of AirdropDashboardScreen.tsx

Reply:
use walletsSelectors.wallets

Reply:
done